pkg/api.go

```diff
@@ -18,7 +18,7 @@ type Endpoint struct {
 type JSONTime time.Time
 
 func (t JSONTime) MarshalJSON() ([]byte, error) {
-	stamp := fmt.Sprintf("\"%s\"", time.Time(t).Format("2006-01-02 15:04:05"))
+	stamp := fmt.Sprintf("\"%s\"", time.Time(t).Format("2006-01-02 15:04:05.001"))
```

Comment:
Upped the time resolution to prevent duplicate keys in `checkSets`.
Also logged https://github.com/flanksource/canary-checker/issues/105 for **http** checks with same **endpoint** with different **responseCodes** that cause [duplicate keys](https://github.com/philipstaffordwood/canary-checker/blob/5708ccdbd5ec74859c522830b83bcd12c344f38e/statuspage/src/store/index.js#L29)